src_tests/common_code_for_tests.py

```diff
@@ -0,0 +1,50 @@
+__author__ = 'Tomasz Rybotycki'
+
+from numpy import zeros
+from typing import List
+
+from src.LossyBosonSamplingExactDistributionCalculators import BosonSamplingExperimentConfiguration
+from src.BosonSamplingSimulator import BosonSamplingSimulator
+from src.simulation_strategies.SimulationStrategy import SimulationStrategy
+from src.Boson_Sampling_Utilities import generate_possible_outputs
+
+
+class ApproximateDistributionCalculator:
+    def __init__(self, experiment_configuration: BosonSamplingExperimentConfiguration,
+                 strategy: SimulationStrategy):
+        self.configuration = experiment_configuration
+        self.strategy = strategy
+
+    def calculate_approximate_distribution(self, samples_number: int = 5000) -> List[float]:
+        """
+            Prepares the approximate distribution using boson sampling simulation method described by
+            Oszmaniec and Brod. Obviously higher number of samples will generate better approximation.
+            :return: Approximate distribution as a list.
+        """
+        possible_outcomes = generate_possible_outputs(self.configuration.number_of_particles_left,
+                                                      self.configuration.number_of_modes)
+
+        simulator = BosonSamplingSimulator(self.configuration.number_of_particles_left,
+                                           self.configuration.initial_number_of_particles,
+                                           self.configuration.number_of_modes, self.strategy)
+
+        outcomes_probabilities = zeros(len(possible_outcomes))
+
+        for i in range(samples_number):
+
+
+            # print(f'Sample number {i} of {samples_number}.')
```

Comment:
Commented out `print` as well as large empty area. Would you mind reformatting this file to `PEP-8` standard, please?

Reply:
It's no longer here (pull request made from old code ^^').